### PR TITLE
Update README.MD with prometheus.

### PR DESCRIPTION
**Pull request by rukawata**

### CODE REVIEW

**Comment by secustor** [comment]
Could you add the `prometheus_enabled` option to the examples in this PR too? 

We have run in this issue today.

**Comment by rukawata** [comment]
> Could you add the `prometheus_enabled` option to the examples in this PR too?
> 
> We have run in this issue today.

Thanks for your feedback! We added that to all the samples we provided under `deploy/v2/template_samples`.

**Comment by ben-moss** [comment]
I found this back in #287. It really should be defaulted in ansible to prevent future templates having the same problem. FWIW @robfallows also ran into the same problem today.

**Comment by rukawata** [comment]
> I found this back in #287. It really should be defaulted in ansible to prevent future templates having the same problem. FWIW @robfallows also ran into the same problem today.

Yeah. The challenge here is also how and where to maintain all the variables in the input JSON (terraform and ansible share the same variable file aka input JSON).
Since terraform always do a replace instead of merge if the variable is map or object, we can't rely on terraform to take consolidate the customized value with default value here.

We have this [template](https://github.com/Azure/sap-hana/blob/master/templates/tempGen/template-daily.json) used in our daily test, and this [template](https://github.com/Azure/sap-hana/blob/master/templates/tempGen/template.json) used in our PR. Those templates are guaranteed to run fine. So my thinking is I will point to those templates in the README.MD for people who are new to the script to try it out.

Hope this helps.

Thanks,
Nancy

File: deploy/v2/README.MD [hunk 1]
```diff
@@ -241,6 +242,8 @@ And we only include OS configuration for:
 - SLES 12
 - RHEL 7.
 
+<sup>[2](#myfootnote2)</sup>: With [Prometheus](https://prometheus.io/) enabled, you will have more possiblity when you monitor your HANA instance using Azure Monitor for SAP Solutions (preview).
```
Comment (persiaAziz) [inline deploy/v2/README.MD]: nit: enabling promethus will enrich the set of telemetry even more when you monitor your Hana Instance using Azure Monitor for SAP Solutions

Comment (rukawata) [reply deploy/v2/README.MD]: Thanks! Addressed this comment.

**Pull request merged**